MAINTAINERS.yml

```diff
@@ -2926,6 +2926,28 @@ Networking:
   tests:
     - sample.net.ptp
 
+"Networking: Prometheus":
+  status: maintained
+  maintainers:
+    - mustafaabdullahk
+  collaborators:
+    - rlubos
+    - jukkar
+  files:
+    - include/zephyr/net/prometheus/*.h
+    - subsys/net/lib/prometheus/
+    - samples/net/prometheus/
+    - tests/net/lib/prometheus/
+  labels:
+    - "area: Prometheus"
+  tests:
+    prometheus.collector
```

Comment:
Wouldn't a single `- prometheus` entry enought to cover all test subcategories? Missing `-` also seems to be the case for the failing CI.

Reply:
Yes you'r right. Wrote to similiar entry but i removed and just keep 'prometheus'.